engine/build.gradle

```diff
@@ -132,6 +132,9 @@ dependencies {
 
     // In addition to all the above the dev source set also needs to depend on what gets compiled in main
     devCompile sourceSets.main.output
+
+    // Dependency on CrashReporter
+    compile group: 'org.terasology.crashreporter', name: 'cr-terasology', version: '4.0.0'
```

Comment:
Can you try using the fancier setup method from https://github.com/MovingBlocks/Terasology/blob/develop/facades/PC/build.gradle#L43-L58? Might fix the issue you've mentioned in the... issue 😄